test/test_geometry.jl

```diff
@@ -95,6 +98,25 @@ end
     @test AG.toJSON(point) == "{ \"type\": \"Point\", \"coordinates\": [ 100.0, 70.0 ] }"
 end
 
+
+
+@testset "convert point format" begin
+    point = AG.createpoint(100, 70)
+    json = convert(GFT.GeoJSON, point)
+    kml = convert(GFT.KML, point)
+    gml = convert(GFT.GML, point)
+    wkb = convert(GFT.WellKnownBinary, point) 
+    wkt = convert(GFT.WellKnownText, point) 
+    @test GFT.val(json) == AG.toJSON(point)
+    @test GFT.val(kml) == AG.toKML(point)
+    @test GFT.val(gml) == AG.toGML(point) # This isn't actually tested above
+    @test GFT.val(wkb) == AG.toWKB(point)
+    @test GFT.val(wkt) == AG.toWKT(point)
+    @test convert(GFT.GeoJSON, json) == json
+    @test convert(GFT.GeoJSON, wkb) == convert(GFT.GeoJSON, wkt) == convert(GFT.GeoJSON, gml)
+    @test convert(GFT.KML, gml) == convert(GFT.KML, wkt)
```

Comment:
Can we move all the conversion tests into its own testset? So maybe something like

```julia
@testset "format conversions" begin
    @testset "point geometry" begin
        ...
    end

    @testset "complex geometries" begin
        ...
    end
end
```

Reply:
if we have a conversions.jl file we could have a test_converstions.jl to test them in